### PR TITLE
Improve Inspector plugin tutorial with a better example

### PR DESCRIPTION
**Pull request by pycbouh**
The example given in this article currently implements a control that looks and acts exactly as the default one. This can hinder the experience of developers trying to follow the guide as it doesn't give them a clear indication if the plugin is working or not.

I've created an alternative example instead and extended the article as well to give more details on how it works. Some parts of it were described in the comments of code samples, which makes them untranslatable.

### CODE REVIEW

File: tutorials/plugins/editor/inspector_plugins.rst [hunk 1]
```diff
@@ -4,125 +4,178 @@ Inspector plugins
...
     var plugin
 
 
     func _enter_tree():
-        # EditorInspectorPlugin is a resource, so we use `new()` instead of `instance()`.
-        plugin = preload("res://addons/MyPlugin/MyInspectorPlugin.gd").new()
+        plugin = preload("res://addons/my_inspector_plugin/MyInspectorPlugin.gd").new()
         add_inspector_plugin(plugin)
-
+
```
Comment (akien-mga) [inline tutorials/plugins/editor/inspector_plugins.rst:52]: Trailing whitespace.

**Comment by pycbouh** [comment]
Trimmed and rebased.

**Comment by pycbouh** [comment]
Any chance this can be reviewed? 🙂

**Comment by NathanLovato** [comment]
It's looking good at first glance, mostly needs some quick proofread and edits. May I add a commit directly on top of your PR (it's more efficient than leaving you review comments)?

**Comment by pycbouh** [comment]
@NathanLovato Of course, if you think it's the most efficient way.

**Comment by NathanLovato** [comment]
Thanks!

**Pull request merged**